ipaserver/install/server/replicainstall.py

```diff
@@ -1077,7 +1099,7 @@ def promote_check(installer):
     remote_api.finalize()
     installer._remote_api = remote_api
 
-    check_remote_version(remote_api)
+    check_remote_compatibility(remote_api)
```

Comment:
Yes, that's what happens now with the `is_fips_enabled()` guards. Since FIPS is not available on DL0, it should fail.